Sources/Sentry/SentryUIPerformanceTracker.m

```diff
@@ -0,0 +1,239 @@
+#import "SentryUIPerformanceTracker.h"
+#import "SentryHub.h"
+#import "SentryLog.h"
+#import "SentryPerformanceTracker.h"
+#import "SentrySDK+Private.h"
+#import "SentryScope.h"
+#import "SentrySpanId.h"
+#import "SentrySwizzle.h"
+#import "SentryUIViewControllerSanitizer.h"
+#import <objc/runtime.h>
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+#endif
+
+static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID
+    = @"SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID";
+
+static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID
+    = @"SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID";
+
+static NSString *const SENTRY_VIEWCONTROLLER_RENDERING_OPERATION = @"ui.rendering";
+
+@implementation SentryUIPerformanceTracker
```

Comment:
if we decide going to do https://github.com/getsentry/sentry-cocoa/pull/1105/files#r639596143 we may need to rename this class too